pkg/render/intrusion_detection.go

```diff
@@ -1146,14 +1184,252 @@ func (c *intrusionDetectionComponent) intrusionDetectionAnnotations() map[string
 	return c.cfg.TrustedCertBundle.HashAnnotations()
 }
 
-func (c *intrusionDetectionComponent) intrusionDetectionADJobsPodTemplate() []client.Object {
-	trainingJobPodTemplate := c.getBaseIntrusionDetectionADJobPodTemplate(ADJobPodTemplateBaseName + ".training")
-	detecionADJobPodTemplate := c.getBaseIntrusionDetectionADJobPodTemplate(ADJobPodTemplateBaseName + ".detection")
+// AD API RBAC for accessing token and subject access reviews for AD Pod token verification
+func (c *intrusionDetectionComponent) getADAPIServiceAccount() *corev1.ServiceAccount {
```

Comment:
nit: throughout the render package we don't name the methods `getX`, but just `X`.

Reply:
https://github.com/tigera/operator/pull/1931/commits/136b82e64ba9d34bee78baa945ee24909eb56e53 fixed here